components/actiongroup/stories/template.js

```diff
@@ -129,59 +130,24 @@ export const OverflowOption = (context) => Container({
 	`
 }, context );
 
-export const ActionButtonOptions = (args, context ) => Container({
```

Comment:
Correct me if I'm wrong, but I think this was basically recreating the Template with the default args. I ended up removing this template, in favor of just using the template itself in `TreatmentTemplate`.